client/src/components/List/AddListContainer/useStyles.ts

```diff
@@ -0,0 +1,32 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    minHeight: '200px',
+  },
+  dialogPaper: {
+    height: '400px',
+    minHeight: '400px',
+  },
+  authWrapper: {
+    display: 'flex',
+    alignItems: 'flex-start',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    minHeight: '200px',
+    paddingTop: 23,
+  },
+  welcome: {
+    fontSize: 26,
+    paddingBottom: 20,
+    color: '#000000',
+    fontWeight: 700,
+    fontFamily: "'Open Sans'",
+    textAlign: 'center',
+  },
+  closeSignUpButton: {
+    textAlign: 'right',
+  },
```

Comment:
u can add `cursor: 'pointer',` here to make the close button behave like a button